Directory.Packages.props

```
@@ -21,10 +21,10 @@
     <PackageVersion Include="MinVer" Version="5.0.0" />
     <PackageVersion Include="Moq" Version="4.18.4" />
     <PackageVersion Include="morelinq" Version="4.2.0" />
-    <PackageVersion Include="MSTest.TestFramework" Version="3.5.1" />
-    <PackageVersion Include="MSTest.Analyzers" Version="3.5.1" />
-    <PackageVersion Include="MSTest.TestAdapter" Version="3.5.1" />
-    <PackageVersion Include="Microsoft.NET.Test.Sdk" Version="17.10.0" />
+    <PackageVersion Include="MSTest.TestFramework" Version="3.5.2" />
```

You probably need to also add `MSTest` metapackage (it is used here https://github.com/microsoft/component-detection/blob/main/test/Microsoft.ComponentDetection.VerificationTests/Microsoft.ComponentDetection.VerificationTests.csproj#L21)

```suggestion
    <PackageVersion Include="MSTest" Version="3.5.2" />
    <PackageVersion Include="MSTest.TestFramework" Version="3.5.2" />
```

---

good find, removing that ref in favor of `MSTest.TestAdapter` and `Microsoft.NET.Test.Sdk`, similar to other test packages that are not packaged